.github/workflows/behat-stability-1-basic-installation.yml

```diff
@@ -1,9 +1,9 @@
-name: 'Behat'
+name: 'Behat Basic Installation'
```

Comment:
@tbsiqueira Any reasons for changing the name? Please see the current output after this change. 

<img width="864" alt="Screenshot 2022-07-01 at 6 47 33 PM" src="https://user-images.githubusercontent.com/8435994/176902339-ff9373a0-b6dd-47bb-a384-6b70a1e8dad4.png">

I made it consistent be like:

Behat / Basic Installation
Behat / Stability 1

and so on.